lib_mic_array/src/decimator_interface.xc

```diff
@@ -1,10 +1,14 @@
-// Copyright (c) 2015-2017, XMOS Ltd, All rights reserved
+// Copyright (c) 2015-2019, XMOS Ltd, All rights reserved
 #include "mic_array.h"
 #include <xs1.h>
 #include <string.h>
 
 #define XASSERT_UNIT DEBUG_MIC_ARRAY
 
+#define DEBUG_UNIT MIC_ARRAY
+#define DEBUG_PRINT_ENABLE_MIC_ARRAY 1
```

Comment:
Set back to 0?

Reply:
I don't think this should be defined in the implementation at all. Should it not be defined in config header (included before line 8) or the Makefile, so that applications can enable or disable this as required?